IKVM-Ullink.nuspec

```diff
@@ -37,5 +37,7 @@
         <file src="bin\IKVM.Reflection.dll" target="lib\net35\IKVM.Reflection.dll" />
         <file src="bin\IKVM.Runtime.dll" target="lib\net35\IKVM.Runtime.dll" />
         <file src="bin\IKVM.Runtime.JNI.dll" target="lib\net35\IKVM.Runtime.JNI.dll" />
+        <file src="bin-x64\ikvm-native.dll" target="lib\net35\ikvm-native.dll" />
```

Comment:
On ODISYS master we are gonna support only x64, so the dll we have now is only for x64. Therefore we want only the x64 dll for now. After we package it, we are gonna test it against x86 and see how it it behaves. Depending on that, we will include the dll for x86 on the other versions of Odisys.

Reply:
I believe the main question would be, do we need to package both CPU archi ? And how should we build the package, or if we should have two packages ?

The [nuget doc](https://docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks#architecture-specific-folders) is talking about that. Maybe you could check if its apply to our case ?

Another option could be to do something custom like renaming the DLL according to the CPU archi but this is, by definition, not standard and, at usage, it will always be required to rename it.

Reply:
Yes, like Florian says, we need to use the intended way by nuget to deliver those arch-dependent assemblies. If it doesn't work properly with our case then we can use them like this.

Reply:
Done